Classes/Controller/VarnishCacheController.php

```diff
@@ -172,7 +172,7 @@ public function purgeAllVarnishCacheAction(Site $site = null, $contentType = nul
         }
         $service = new VarnishBanService();
         $service->banAll($domains, $contentType);
-        $this->addFlashMessage(new Message(sprintf('All varnish cache cleared for %s%s', $site ? 'site ' . $site->getName() : 'installation', $contentType ? ' with content type "' . $contentType . '"' : '')));
+        $this->addFlashMessage((string) new Message(sprintf('All varnish cache cleared for %s%s', $site ? 'site ' . $site->getName() : 'installation', $contentType ? ' with content type "' . $contentType . '"' : '')));
```

Comment:
I'd suggest to drop the `new Message()` completely so it's just a sprintf like the other flashmessage in `purgeCacheByTagsAction()`.

Reply:
You got it :-)